Portal/src/Datahub.Portal/Startup.cs

```diff
@@ -468,7 +468,6 @@ private void ConfigureCoreDatahubServices(IServiceCollection services)
 
         services.AddSingleton<ServiceAuthManager>();
 
-        services.AddSingleton<ICKANServiceFactory, CKANServiceFactory>();
```

Comment:
I don't think we can remove this if it's still being used by the `OpenDataService`. What do you think of updating that service to instantiate the factory instead?

Reply:
Done. It's legacy so it'll be removed eventually anyway, but for now that'll do.